deployments/auth.yaml

```diff
@@ -99,6 +99,18 @@ rules:
   - '*'
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: network-resources-injector-configmaps
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - '*'
```

Comment:
nit: we could restrict the verbs to `get` only.

Reply:
Done.